src/common/models/Projection.ts

```diff
@@ -223,8 +223,14 @@ export class Projection {
     this.region = region;
 
     // Set up our series data exposed via getDataset().
+    this.caseDensityByCases = metricsTimeseries.map(
+      row => row && row.caseDensity,
+    );
+
     this.rawDailyCases = actualTimeseries.map(row => row && row.newCases);
-    this.smoothedDailyCases = this.smoothWithRollingAverage(this.rawDailyCases);
+    this.smoothedDailyCases = this.denormalizeByPopulation(
+      this.caseDensityByCases,
+    );
```

Comment:
The one "annoying" thing about using the caseDensity metric is that it's normalized by population, which we don't want.  So we have to denormalize it. 🤷